### PR TITLE
Changing the Privacy section on the manage students page

### PR DESCRIPTION
**Pull request by brickware**
Per the [speclet](https://docs.google.com/document/d/1xlkkOnUNXca0cMKIAbREc6ilewlOJdTPf-3mV3C9Vqo/edit?ts=5d1bd169) changed the Privacy section to look like this: 

privacysection

Added 2 new Strings to en_us.json that will need to be translated. 

Note that the Privacy section was and still is part of the ManageStudentsLoginInfo component, but separating that out will be a different task.

### CODE REVIEW

**Comment by brickware** [comment]
UI test failed so I committed a change that hopefully will fix it.  Hopefully the build/test will catch it.

**Review by davidsbailey — APPROVED** [review]
Looks great, nice work @brickware !

**Comment by dju90** [comment]
Looks good to me too! Thanks @brickware !

File: dashboard/test/ui/features/teacher_dashboard/teacher_dashboard.feature [hunk 1]
```diff
@@ -40,8 +40,10 @@ Feature: Using the teacher dashboard
     And I click selector "#uitest-teacher-dashboard-nav a:contains(Manage Students)" once I see it
     And I wait until element "#uitest-manage-students-table" is visible
     And element "#uitest-manage-students-table tr:eq(1)" contains text "Sally"
+    And I wait until element "#uitest-privacy-text" is visible
+    And element "#uitest-privacy-text" contains text "Protecting your students' privacy"
```
Comment (islemaster) [inline dashboard/test/ui/features/teacher_dashboard/teacher_dashboard.feature:44]: Looks like you introduced a reference to an element `#uitest-privacy-text` but we never gave that id to any element.

Comment (brickware) [reply dashboard/test/ui/features/teacher_dashboard/teacher_dashboard.feature:44]: Oh shoot, I forgot to commit that. Will do that now!

**Review by islemaster — APPROVED** [review]
:tada: Looks great!

**Pull request merged**